src/test/java/core/basesyntax/service/impl/ParserServiceImplTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParserService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ParserServiceImplTest {
+    private static final String FRUIT_NAME = "banana";
+    private static final int FRUIT_QUANTITY = 80;
+    private static ParserService parserService;
+    private List<String> dataList;
+
+    @BeforeAll
+    static void beforeAll() {
+        parserService = new ParserServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        dataList = new ArrayList<>();
+    }
+
+    @Test
+    void parse_validData_ok() {
```

Comment:
try follow the naming recomendation in common mistakes, please, fix it everywhere in your code: [naming rule](https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop-tests#remember-about-naming-of-test-methods)